nipype/interfaces/ants/preprocess.py

```diff
@@ -0,0 +1,358 @@
+# -*- coding: utf-8 -*-
+"""The ants module provides basic functions for interfacing with ants
+    functions.
+"""
+import csv
+import math
+import os
+
+from ..base import (BaseInterface, BaseInterfaceInputSpec, TraitedSpec, File,
+                    traits, isdefined, Str)
+from .base import ANTSCommand, ANTSCommandInputSpec
+from ...utils.filemanip import split_filename
+
+
+def _extant(field):
```

Comment:
Is this really necessary? Since these are `File` traits, you can't set `None` on them. Why not using `isdefined` exclusively?